react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+
+export function Accordion({ array }) {
```

Comment:
The name `array` doesn't convey much information. It would be better to have a prop name that describes its purpose.